pyprof/tests/L0/run_test.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
```

Comment:
if this change is unrelated to this PR, can it be removed and saved for a separate PR?